src/index.ts

```diff
@@ -0,0 +1,34 @@
+import fastify from "fastify";
+import { readdirSync } from "fs";
+import { prod } from "./common/constants";
+
+export const server = fastify({
+    logger: {
+        development: {
+            transport: {
+                target: "pino-pretty",
+                options: {
+                    translateTime: "HH:MM:ss Z",
+                    ignore: "pid,hostname"
+                }
+            }
+        },
+        production: false
```

Comment:
style: trailing comma on multiline objects for better diffs
```suggestion
        production: false,
```